system-tests/package.json

```diff
@@ -90,5 +91,14 @@
   "types": "lib/fixtures.ts",
   "license": "ISC",
   "author": "",
-  "keywords": []
+  "keywords": [],
+  "nx": {
+    "targets": {
+      "build": {
+        "dependsOn": [
+          "!@packages/data-context:build"
```

Comment:
No idea why I had to add this. Nx was saying that `@tooling/system-tests:build` relied on `@packages/data-context:build` which was causing a circular dependency, but I couldn't figure out why it thought that - if you look at the dep graph, `@tooling/system-tests` doesn't depend on `@packages/data-context` so I don't understand